carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/DataPointArchiveRocksDB.java

```diff
@@ -130,20 +130,16 @@ class DataPointArchiveRocksDB
         this.catchUpTimer = metricRegistry.timer(MetricUtils.dbCatchUpTimerName(dbName));
         this.catchUpTimerError = metricRegistry.meter(MetricUtils.dbCatchUpTimerErrorName(dbName));
         this.longId = longId;
-        if (rocksdbConfig.readOnly) {
```

Comment:
just curious, why are we making this change?

Reply:
This is a flip to original code, because I find out that even in readonly mode, we need the cleaner.